src/containers/Userlist.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import { Select, MenuItem, Alert, AlertTitle } from '@mui/material';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { defaultProfilePic, lock, unlock, loading } from '../assets';
+import {
+  disableUser,
+  enableUser,
+  reset
+} from '../states/features/users/usersSlice';
+import Loading from '../components/Loading';
+
+export const Userlist = ({ userList }) => {
+  const dispatch = useDispatch();
+  const { isLoading, accountStatus, message } = useSelector((state) => {
+    return state.users;
+  });
+  const [open, setOpen] = React.useState(false);
```

Comment:
why `React.useState` yet you already imported useState. You only do that if you didn't distructure useState in the import